src/esp/physics/PhysicsManager.cpp

```diff
@@ -68,7 +70,8 @@ int PhysicsManager::addObject(const int objectLibIndex,
   //! Draw object via resource manager
   //! Render node as child of physics node
   resourceManager_->loadObject(
-      configFile, &existingObjects_.at(nextObjectID_)->node(), drawables);
+      configFile, existingObjects_.at(nextObjectID_)->visualNode_, drawables);
+  existingObjects_.at(nextObjectID_)->node().computeCumulativeBB();
```

Comment:
how do you feel about moving https://github.com/facebookresearch/habitat-sim/blob/01a8e36ba8fba7192e7a336a0b26b3b35df95603/src/esp/assets/ResourceManager.cpp#L448 here too? This is more a responsibility of physics manager anyways

Reply:
Thanks for point this out. Actually, that call is redundant after this addition.